default-config.yml

```diff
@@ -13,44 +13,31 @@ receivers:
     protocols:
       grpc:
       http:
-
   opencensus:
-
-  # Collect own metrics
-  prometheus:
-    config:
-      scrape_configs:
-        - job_name: "otel-collector"
-          scrape_interval: 10s
-          static_configs:
-            - targets: ["0.0.0.0:8888"]
-
   jaeger:
     protocols:
       grpc:
       thrift_binary:
       thrift_compact:
       thrift_http:
-
   zipkin:
 
 processors:
   batch:
 
 exporters:
-  logging:
-    logLevel: debug
+  kafka:
+    protocol_version: 2.0.0
+    brokers:
+      - kafka-zookeeper:9092
+    topic: jaeger-spans
```

Comment:
Looks very specific config as default, and if someone wants to use our collector out-of-box, would make it difficult?

Reply:
Can config folders can have more than one config files as part of an image? And docker-compose setup can use the one which requires for HT. So, that way, it can work in default as well as along with HT bypassing config argument.

Reply:
Done

Reply:
I am not sure about the point you make @kotharironak. I feel like in most of the cases people will use the collector with hypertrace platform and in some cases people might try this collector because of the PII filter we are adding and in that case they need to touch the config.

Reply:
If, that is the case, should be fine. But, the default config will still be very specific to docker-compose setup
- kafka-zookeeper (we still have to override with helm deployment). So, we can change it to `bootstrap`, and add one more alias in docker-compose.
- jaeger-spans  (this should be fine, and can be considered as our default entry point)

Reply:
@kotharironak , this configs are not gonna affect helm in any way. there's separate configmap for helm.

Reply:
> kafka-zookeeper (we still have to override with helm deployment). So, we can change it to bootstrap, and add one more alias in docker-compose.

As per https://github.com/open-telemetry/opentelemetry-collector/commit/047b0f38e6816b12ffff10fc276eee032f4ad987 we can use env var for that I think.

Reply:
@jcchavezs , I made ht-config as default while passing in command.